docs/content/distribute-bundles.md

```diff
@@ -90,7 +95,18 @@ porter publish -a mybunz1.1.tgz --tag getporter/megabundle:1.1.0
 
 ## Image References After Publishing
 
-When a bundle is published, the images that it will use are copied into the location of the published bundle. This simplifies access control and management of artifacts in the repository. Consider the following `porter.yaml` snippet:
+When a bundle is published, all images [referenced][image-map] by the bundle are
+published **inside** bundle repository. Bundles should be written to use the
```

Comment:
I don't really follow what is the action a bundle author should take from this.
Is this part not covered by relocation maps?

Reply:
There is another [PR](https://github.com/deislabs/porter/pull/1042) where I've been working on building up docs for how to actually use the relocation map properly in your bundle. For example just populating the `images` section isn't enough, you also then have to use helm charts that understand image digests (not tags which they all use) and then actually use the relocation map to get the final location of your images. This is something that porter helps with.

That PR is being held up by this bug at the moment. So I can explain more and then link to that once this is merged. I'll make a note of it on that PR?